functions/go/render-helm-chart/main.go

```diff
@@ -89,7 +89,7 @@ func (f *helmChartInflatorFunction) Config(rn *kyaml.RNode) error {
 			return err
 		}
 	default:
-		return fmt.Errorf("`functionConfig` must be `ConfigMap` or `InflateHelmChart`")
+		return fmt.Errorf("`functionConfig` must be `ConfigMap` or `RenderHelmChart`")
```

Comment:
Can you define `ConfigMap` and `RenderHelmChart` as const?

Reply:
done